src/bookingRepository.js

```diff
@@ -0,0 +1,26 @@
+class BookingRepository {
+  constructor(bookings) {
+    this.bookings = bookings;
+  }
+
+  viewBookings(guestId) {
+    return this.bookings.filter(booking => booking.userID === guestId)
+  }
+
+  makeBooking(guestId, date, room) {
```

Comment:
Awesome BookRepo class, I noticed that this method returns an object of what the bookings look like, I may be way off but I think that if you use this template to do the Post request, you might get errors because the post format is an object of just 3 key-value pairs. But, if I'm way off and youre not using this for you Post request, then it looks really solid!

Reply:
You are 100% right - THANK YOU! Saved me lots of time :) I need to read the spec directions more carefully.